lib/release-ops.js

```diff
@@ -249,6 +249,33 @@ function calculateReleaseFromGitLogs(currentVersion, logs, prereleaseId) {
     return releaseInfo;
 }
 
+/**
+ * Gets the range of commits to include in a changelog.
+ * If this will be the first stable release following a prerelease sequence,
+ * all commits going back to the previous stable release are included.
+ * @param {string[]} tags All prior version tags.
+ * @param {string} [prereleaseId] The prerelease identifier if this is a prerelease.
+ * @returns {string} The commit range to include in the changelog.
+ * @private
+ */
+function getChangelogCommitRange(tags, prereleaseId) {
+    let lastTag = null;
+
+    // If this will be a stable release after a prerelease...
+    if (!prereleaseId && semver.prerelease(tags[tags.length - 1])) {
+        let i = 2;
+
+        do {
+            lastTag = tags[tags.length - i];
+            i++;
+        } while (semver.prerelease(lastTag));
+    } else {
+        lastTag = tags[tags.length - 1];
+    }
+
+    return lastTag ? `${lastTag}..HEAD` : "";
+}
+
 /**
  * Gets all changes since the last tag that represents a version.
```

Comment:
Should we update this description?

Reply:
Good catch. Updated in a75dc14. One downside to only combining prereleases' changelogs for the next stable release's changelog is that it's harder to explain than "changelogs include all changes since the last stable release."

Reply:
That’s true. I can add that to post - feel free to give feedback!